cscope.py

```diff
@@ -295,24 +334,31 @@ def __init__(self, view, platform, database, symbol, mode, executable):
     # switch statement for the different formatted output
     # of Cscope's matches.
     def append_match_string(self, match, command_mode, nested):
-        match_string = "{0}".format(match["file"])
+        match_string = {}
+        match_string['main'] = "{0}".format(match["file"])
         if command_mode in [0, 4, 6, 8]:
             if nested:
-                match_string = ("{0:>6}\n{1:>6} [scope: {2}] {3}").format("..", match["line"], match["scope"], match["instance"])
+                match_string['head'] = ("{0:>6}\n").format("..")
+                match_string['main'] = ("{0:>6} [scope: {1}] {2}").format(match["line"], match["scope"], match["instance"])
```

Comment:
Yeah, going to need help understand what you're doing here too, please, in each of these. I didn't write these regexps originally, so I honestly don't understand what they were doing completely nor what you're changing.

Reply:
The regexes just format results appearance. I only split them into the 'head' part with the separator ('..') or file path and 'main' part with proper result line, which was needed for keyboard navigation and selecting/highlighting. Effectively there is no difference in appearance.

I would also add the option to the settings for disabling the separator between results altogether.